kuma/wiki/jinja2/wiki/document.html

```diff
@@ -356,6 +356,14 @@ <h2 class="offscreen">{{ _('Document Tags and Contributors') }}</h2>
                   {% endif %}
                 </div>
               {% endif %}
+
+              {% if waffle.switch('newsletter_article') %}
```

Comment:
Should this be ``waffle.switch('newsletter') and waffle.switch('newsletter_article')`` like the JS include?